bld_rights_db.pl

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
```

Comment:
The default perl in our docker image is `/usr/local/bin/perl`. This invocation will get the correct default perl no matter our environment. I am not sure whether it makes more sense to use the Debian system perl (as we do in https://github.com/hathitrust/feed) or the official perl docker image (as we're doing here.)

Reply:
I guess my knee-jerk take is that we rely on a lot of system plumbing beyond Perl in most of our images, so it adds less cognitive load to standardize on "whatever Debian provides". Compare and contrast something like Otis, where Ruby is front and center and system services are relatively in the background. That's how I rationalize the situation, anyway.

I was informed long ago that you can't go wrong with `/usr/bin/env progname`